js/dom.js

```diff
@@ -68,25 +68,24 @@ onDOMready().then(() => {
   $.remove('#firefox-transitions-bug-suppressor');
   initCollapsibles();
   focusAccessibility();
+  if (!chrome.app && chrome.windows) {
+    // add favicon in Firefox
+    setTimeout(() => {
+      if (!prefs) {
+        return;
+      }
+      const iconset = ['', 'light/'][prefs.get('iconset')] || '';
```

Comment:
Since the prefs is async initialized, we have to wait till the initialization complete.